Handlers/Chat.py

```diff
@@ -51,7 +51,11 @@ def insert_chat(self, data):
             chat_name = data['chat_name']
             username = get_jwt_identity()
             user = self.userDAO.get_user_by_username(username)
-            cid = self.chatDAO.insert_chat_group(chat_name, user['uid'])
+            if data['members'] is not None:
```

Comment:
May produce a key error. Must first check if key is in dictionary

Reply:
Simpre va a estar en el dictionary, eso es lo que hace es parse

Reply:
Si no esta en el request pues lo pone como None